build/cmake/DefineInstallationPaths.cmake

```diff
@@ -23,4 +23,9 @@ set(BIN_INSTALL_DIR "bin" CACHE PATH "The binary install dir (default: bin)")
 set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}" CACHE PATH "The library install dir (default: lib${LIB_SUFFIX})")
 set(INCLUDE_INSTALL_DIR "include" CACHE PATH "The library install dir (default: include)")
 set(CMAKE_INSTALL_DIR "cmake" CACHE PATH "The subdirectory to install cmake config files (default: cmake)")
+set(PKGCONFIG_INSTALL_DIR "lib/pkgconfig" CACHE PATH "The subdirectory to install pkgconfig config files (default: lib/pkgconfig)")
 set(DOC_INSTALL_DIR "share/doc" CACHE PATH "The subdirectory to install documentation files (default: share/doc)")
+set(prefix "${CMAKE_INSTALL_PREFIX}")
```

Comment:
Could we move these into ADD_PKGCONFIG_THRIFT so they are not global?

Reply:
That seems to work.

Cmake variables are scoped to the current directory when they are defined in by default, rather than anything related to the function/macro they are defined in though.

Reply:
Hmm, I may be confused. Apparently functions do introduce a new scope. I'm honestly not clear how this works, but I think what you suggest is fine.

Reply:
Items in DefineInstallationPaths.cmake are global.  I'm probably just being pedantic about this - those names are fairly generic.